code/modules/assembly/flash.dm

```diff
@@ -200,17 +200,34 @@
 
 /obj/item/assembly/flash/cyborg/attack(mob/living/M, mob/user)
 	..()
-	new /obj/effect/temp_visual/borgflash(get_turf(src))
+	if(cooldown)
+		return FALSE
+	else
+		new /obj/effect/temp_visual/borgflash(get_turf(src))
+		return 
 
 /obj/item/assembly/flash/cyborg/attack_self(mob/user)
 	..()
-	new /obj/effect/temp_visual/borgflash(get_turf(src))
+	if(cooldown)
+		return FALSE
+	else
+		new /obj/effect/temp_visual/borgflash(get_turf(src))
+		return 

```

Comment:
revert every change in this file above this line.

Reply:
overwrite both attack and attack_self with the following

```DM
if(..())
    new /obj/effect/temp_visual/borgflash(get_turf(src))

Reply:
Sorry I don't really understand what I should do here?

Reply:
ok ill try to better display what I want.